### ISSUE TITLE
How do I reload the table when a new row is added?

### ISSUE THREAD

**Issue by LibbyJax** [markcell/jQuery-Tabledit#9]
Is it possible to reload the plugin on a table when a new row is added?  Or is there a remove function and then I can reload it fully?

**Comment by markcell** [comment]
Not in the plans, because it will duplicate things. But it's a good idea to create a method to destroy and reload.

**Comment by structure-pm** [comment]
I agree adding a new row is pretty important for a lot of real-world cases -- if a row can be deleted, why not added?

Great project though. Lightweight and functional for straightforward use.

**Comment by sarla999** [comment]
I think if you can add new row . You can try add a default data to db , then you edit the new row :)

**Comment by nanexcool** [comment]
Any updates on this? I'm adding rows dynamically to a table and it would be great to have Tabledit parse the table on the fly!

**Comment by Rizver** [comment]
Hello

It's a great plugin to use! I just have a very small question 

when a row is deleted or edited it is set to $lastDeletedRow or $lastEditedRow ..

How do I access it ? can you please give me an example?

Thanks in advance!

**Comment by benftwc** [comment]
Hey, this still not implemented, @nanexcool have you create a patch so ? ^^'

**Comment by m2jobe** [comment]
@benftwc Hey did you find a work around for this?

**Comment by LibbyJax** [comment]
This is how I'm currently working around it.  After adding a new row to the database, I then make an ajax call to fresh the HTML of the table and call the Tabledit init again.

$("#table-account-mapping-add-new").on('click', function() {
$.ajax({
		            type: "GET",
		            url: "/add_new",
		            cache: false,
		            success: function(data, status) {
		            	$('#table-account-mapping').html(data);
						$('#table-account-mapping').Tabledit({
							url: '/updated_table_html',
							columns: {
								identifier: [0, 'id'],
								editable: [[1, 'account_number'], [2, 'directory_path']]
							},
							hideIdentifier: true
						});
		            },
	                error: function() {
	                }

**Comment by markcell** [comment]
Thanks for your contribution, but since this project hasn't been actively maintained for quite some time and the repository will be archived, I'm closing this issue.
